samples/emoji-search/presenters/build.gradle.kts

```
@@ -4,11 +4,14 @@ apply(plugin = "app.cash.zipline")
 
 plugins {
   kotlin("multiplatform")
+  id("com.android.library")
   kotlin("plugin.serialization")
 }
 
 kotlin {
-  jvm()
+  android {
+    publishAllLibraryVariants()
+  }
```

```suggestion
  android()
```
This isn't published